acceptance/framework/vault/vault_cluster.go

```diff
@@ -147,13 +147,6 @@ func (v *VaultCluster) bootstrap(t *testing.T, vaultNamespace string) {
 	})
 	require.NoError(t, err)
 
-	// Enable the PKI Secrets engine.
-	err = v.vaultClient.Sys().Mount("pki", &vapi.MountInput{
```

Comment:
This has moved to `ConfigurePKI()` in the above helpers file because we will need to mount more than one PKI engine and it will not always be at `pki` base path.